.code-samples.meilisearch.yaml

```diff
@@ -620,10 +620,30 @@ get_all_tasks_paginating_2: |-
   $taskQuery = (new TasksQuery())->setLimit(2)->setFrom(8));
   $client->getTasks($taskQuery);
 get_pagination_settings_1: |-
+  $client->index('books')->getPagination();
 update_pagination_settings_1: |-
+  $client->index('books')->updateSettings([
+    'pagination' => [
+      'maxTotalHits' => 100
+    ]
+  ]);
 reset_pagination_settings_1: |-
+  $client->index('books')->resetPagination();
 get_faceting_settings_1: |-
+  $client->index('books')->getFaceting();
 update_faceting_settings_1: |-
+  $client->index('books')->updateFaceting([ 'maxValuesPerFacet' => 2 ]);
```

Comment:
I didn't find aby test for this method in the SDK `updateFaceting`, did I miss something?

Reply:
Yep, we have!

https://github.com/meilisearch/meilisearch-php/blob/89bcb6387883cd29deeb05aa57e74bc3d4f2915e/src/Endpoints/Delegates/HandlesSettings.php